must-gather/collection-scripts/namespace

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+function namespace() {
+  ns=$( oc get pods -A | awk '/performance-operator/ { print $1 }' )
```

Comment:
Just another way :)

oc get pods -A -l name=performance-operator -o=jsonpath='{.items[0].metadata.annotations.olm\.operatorNamespace}{"\n"}'

Reply:
yours is better (less moving parts), thanks!
And this reminds me I need to improve my oc-command-fu.